src/include/wiredtiger.in

```diff
@@ -1261,6 +1261,15 @@ struct __wt_session {
 	    const char *uri, const char *newuri, const char *config);
 
 	/*!
+	 * Reset the session. Any resources held by the session are released.
+	 * @snippet ex_all.c Reset the session
```

Comment:
The above two lines aren't consistent with other comments. The form is more generally:

* One line with a very brief description, i.e: ```Reset the session.``` (I'd add ```handle``` to be more consistent with other APIs).
* A blank line, followed by the more detailed description of what the API does.
* Another blank line prior to the ```@snippet``` comment.